workshop/content/advanced/resource-types/prerequisites/_index.md

```diff
@@ -14,18 +14,49 @@ Before proceeding to the next section, choose to install the following prerequis
   - [AWS SAM CLI on Windows](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install-windows.html)
   - [AWS SAM CLI on macOS](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install-mac.html)
 
-* install the [CloudFormation Command Line Interface (CLI)](https://docs.aws.amazon.com/cloudformation-cli/latest/userguide/what-is-cloudformation-cli.html) and plugin(s) for supported languages that you wish to use. The following example uses `pip` to install the CloudFormation CLI and language plugins (you can also choose to install the CloudFormation CLI with `brew` and language plugins with `pip` as shown on this [page](https://docs.aws.amazon.com/cloudformation-cli/latest/userguide/what-is-cloudformation-cli.html)):
-
-```
-pip install cloudformation-cli cloudformation-cli-java-plugin cloudformation-cli-go-plugin cloudformation-cli-python-plugin cloudformation-cli-typescript-plugin
-```
+  For example, installing AWS SAM CLI and Docker on macOS using [Homebrew package manager](https://docs.brew.sh/).
```

Comment:
I do recommend keeping the existing section, so to point to the documentation walkthrough for each of the 3 operating systems (including the installation of Docker), and leave as well the existing one-liner for installing dependencies with `pip`.  This is for consistency with [this page](https://cfn101.workshop.aws/prerequisites/awscli.html) of the workshop that also points to the documentation.  I think this approach scales better and is more maintainable.